pilot/lib/pilot/build_manager.rb

```diff
@@ -389,15 +389,24 @@ def expire_previous_builds(build)
     def transporter_for_selected_team(options)
       # Use JWT auth
       api_token = Spaceship::ConnectAPI.token
+      api_key = if options[:api_key].nil? && !options[:api_key_path].nil?
+                  # Load api key info if user set api_key_path, not api_key
+                  { key_id: api_token.key_id, issuer_id: api_token.issuer_id, key: api_token.key_raw, is_key_content_base64: api_token.is_key_content_base64 }
+                elsif !options[:api_key].nil?
+                  options[:api_key]
+                else
+                  nil
```

Comment:
This is the redundant `else` that the linter is warning your about in [this CI run](https://app.circleci.com/pipelines/github/fastlane/fastlane/4827/workflows/f3ddde11-9de4-4d64-990f-5d70a2184212/jobs/85161?invite=true#step-107-105)